src/pydap/tests/test_model.py

```diff
@@ -176,10 +176,7 @@ def test_StructureType_init():
 def test_StructureType_instance():
     """Test that it is a Mapping and DapType."""
     var = StructureType("var")
-    try:
-       from collections.abc import Mapping
-    except ImportError:
-       from collections import Mapping
+    from collections.abc import Mapping
```

Comment:
Nit: It would be a little tidier if this import could be just moved up with the main import statements at the top of this module.